pkg/cmd/roachtest/github.go

```diff
@@ -332,24 +310,42 @@ func (g *githubIssues) createPostRequest(
 		Artifacts:               artifacts,
 		SideEyeSnapshotMsg:      sideEyeMsg,
 		SideEyeSnapshotURL:      sideEyeTimeoutSnapshotURL,
-		ExtraParams:             clusterParams,
+		ExtraParams:             params,
 		HelpCommand:             generateHelpCommand(testName, issueClusterName, roachtestflags.Cloud, start, end),
 	}, nil
 }
 
 func (g *githubIssues) MaybePost(
 	t *testImpl, l *logger.Logger, message string, sideEyeTimeoutSnapshotURL string,
 ) (*issues.TestFailureIssue, error) {
+	params := g.getParameters(t, roachtestutil.UsingRuntimeAssertions(t), t.goCoverEnabled)
 	skipReason := g.shouldPost(t)
 	if skipReason != "" {
 		l.Printf("skipping GitHub issue posting (%s)", skipReason)
+
+		// If we're skipping issue posting, we still want to log the parameters
```

Comment:
Question around this logic: I think I would find it confusing if sometimes I find this log and other times not (when no issue was created). Would it make sense to lift this out as part of a test's regular logging and always log it? It's relatively small so I don't see the harm in having 'duplicate' data, but I might be missing some other context here.

Reply:
That sounds reasonable, I'll change it to unconditionally log the parameters.

Reply:
Sounds reasonable, I've lifted it to be part of the test runner and it now unconditionally logs the parameters on failure.